helm/kafka/scripts/create-keystore.sh

```diff
@@ -0,0 +1,44 @@
+set -e
+
+
+dir=/timecard-restapi-keystore
+alias=$1
+days=100
+password=$2
+bootstrap=$3
+
+cd $dir
+
+if test -f "$alias.keystore.jks";
+then
+    echo "Keystore already created, exiting ..."
+    exit 0
+fi
+
+# Test Connection
+if
+  openssl s_client -connect $3 -key $alias-key.pem -cert $alias-certificate.pem -brief
```

Comment:
Script would be more readable if `$3` was extracted into a local variable before. e.g. `bootstrap_server_url`

Reply:
Done